src/styles/components/application-preloader/styles.less

```diff
@@ -65,3 +65,16 @@
     width: 100px;
   }
 }
+
+& when (@screen-resolution-2x-enabled) {
+
+  @media only screen and (min-resolution: 2dppx) {
+
+    & when (@application-preloader-enabled) {
+
+      .application-preloader-spinner {
+        background-image: url('../../../img/components/application-preloader/application-preloader-spinner@2x.png');
```

Comment:
🤔  Why can't we use the svg instead of adding a 2dppx spinner image? Please keep in mind that people use the UI on there mobile phones (many of which feature retina screens) and this image unnecessarily increases the load time.

Reply:
We can't generate an angular gradient using web-friendly SVGs :/ Unless you know how? So, we're left with a PNG.  That said, I ran the images through ImageOptim and was able to shave off ~70% of the original file size.

Reply:
it looks like there is a way to fake it: http://wiki.inkscape.org/wiki/index.php/Advanced_Gradients#Conical_gradient

IMHO we should try to insert it as SVG instead of a referencing an external image.

Reply:
@juliangieseke I think the file is small enough and might even perform better (rendering) than a _faked_ conical gradient.

Reply:
I was thinking about the extra request - so yes, use whatever is smaller in size and, if its the png, include id as base64 in the HTML. 👍